superset/security/manager.py

```diff
@@ -76,6 +78,13 @@ def __init__(self, **kwargs):
 PermissionViewModelView.list_widget = SupersetSecurityListWidget
 PermissionModelView.list_widget = SupersetSecurityListWidget
 
+# Limiting routes on FAB model views
+UserModelView.include_route_methods = CRUD_ROUTE_METHODS | {"userinfo"}
```

Comment:
Evaluate including `show`, contains helpful admin/audit info about the user

Reply:
Same here. I'd recommend restricting this info unless it's required for application functionality.